app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/DataCollectionFragment.kt

```diff
@@ -170,6 +181,7 @@ class DataCollectionFragment : Fragment() {
             when (destination.id) {
                 R.id.identifyResultFragment -> {
                     toolbar.title = dataCollectionViewModel.portalItemTitle.value
+                    popupViewModel.cancelEditing()
```

Comment:
Were you going to remove this in a subsequent PR?

Reply:
We will need to handle the canceling editing of the popup when the user presses the back button while editing the popup. 
I intend to address this in the upcoming PR where I am cleaning up the UI and polishing some patterns.